<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>MapMySections Data Challenge</title>
    <style>
        body {
            font-family: sans-serif;
            margin: 0;
            display: flex; /* Use flexbox for layout */
			line-height: 1.6; /* Adjust as needed */
        }

        .sidebar {
            width: 2200px; /* Adjust width as needed */
            background-color: #B3C1CD; /* changed from  #f0f0f0; */
            padding: 20px;
            box-sizing: border-box; /* Include padding in width */
            overflow-y: auto; /* Add scroll if content is too long */
            height: 100vh; /* Make sidebar full height */
			border: 1px solid #ccc; /* Example border */
        }

        .main-content {
            flex-grow: 1; /* Allow main content to take remaining space */
            padding: 20px;
            box-sizing: border-box;
            overflow-y: auto; /* Add scroll if content is too long */
            height: 100vh; /* Make main content full height */
			border: 1px solid #ccc; /* Example border */
			color: #003057;
        }

        .sidebar ul {
            list-style: none;
            padding: 0;
        }

        .sidebar li {
            margin-bottom: 10px;
        }

        .sidebar a {
            text-decoration: none;
            color: #003057;
        }
		
		hr {
		    height: 5px; /* Adjust the height */
		    border: none; /* remove default border */
		    background-color: #003057; /* set the color */
		    width: 50%; /* Adjust the width (percentage or pixels) */
		    margin-left: 0;
            margin-right: auto; /* optional, to make it behave more predictably */
		}
    </style>
	
	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css">
	
</head>
<body>

    <div class="sidebar">
        <h2 style="color: #003057;">MapMySections</h2 style="color: #003057;">
		<img src="logo.png" alt="image" width="180px">
        <ul>
            <li><b><a href="#about">1. About this Challenge</a></b></li>
            <li><b><a href="#entry">2. Entry Details</a></b></li>
            <li><b><a href="#data_access">3. Primary image series data access</a></b></li>
            <li><b><a href="#spatial">4. Spatial localization of mouse brain cell types</a></b></li>
            <li><b><a href="#regions">5. Restricting analysis to VISp</a></b></li>
			<li><b><a href="#considerations">6. Key considerations</a></b></li>
			<li><b><a href="#background">7. Background</a></b></li>
			<li><b><a href="#links">8. Useful links</a></b></li>
        </ul>
    </div>

    <div class="main-content">
        <h2 style="color: #003057;" id="about"><i class="fas fa-brain"></i> About this Challenge</h2 style="color: #003057;">
        <p>Transgenic lines and viral tools provide highly valuable resources for targeting subpopulations of cells in the brain of mouse (and other species). The recent identification of &gt;5,000 mouse brain cell populations with distinct gene expression patterns and spatial profiles provides opportunities not only to create new cell type-specific genetic tools, but also to better characterize the cell type specificity of existing genetic tools widely used in neuroscience or newly generated in a lab. Accurately defining this connection between genetic tools and known cell types represents a critical step in interpreting the results of experiments using these tools, from functional assays to potential gene therapies. A detailed cell type characterization of the labeled cells can be achieved through a combination of single cell RNA-sequencing and cell sorting; however, such methods take time, are costly, and are prone to bias. A method for directly inferring cell types from fluorescent images without the need for additional experiments would be immediately applicable to thousands of existing genetic tools, greatly improving their utility and interpretability. <b><em>Entrants are tasked with creating an algorithm that will accurately match fluorescent images to the most likely cell types and/or presenting such an algorithm as part of a user-friendly tool like <a href="https://portal.brain-map.org/atlases-and-data/bkp/mapmycells">MapMyCells</a>.</em></b> Although images span the whole brain, this challenge is focused on defining cell types in primary visual cortex (VISp).</p>
		
		<img src="workflow.png" alt="image" width="800">
		
		<p>This challenge includes images and associated cell type specificity for anonymized genetics tools from the <a href="https://portal.brain-map.org/genetic-tools/genetic-tools-atlas">Genetics Tools Atlas</a>, a searchable web resource representing information and data on enhancer-adeno-associated viruses (enhancer AAVs) and mouse transgenes. Multiple modalities for summarizing data are included as part of the atlas, but only coronal sections collected using Serial Two-Photon Tomography (STPT) sections will be directly included as part of the challenge. Cell type specificity is assessed by applying single cell RNA-sequencing (SMART-Seq v4) on fluorescently labeled cells from each genetic tool, and then mapping these cells to the published taxonomy of cell types in whole mouse brain <a href=https://pmc.ncbi.nlm.nih.gov/articles/PMC10719114/>(Yao et al, 2023)</a>, which are available in the <a href="https://portal.brain-map.org/atlases-and-data/bkp/abc-atlas">Allen Brain Cell Atlas</a>. Qualitative assessments of cell labeling patterns are also provided for many of the genetic tools.</p>

        <br><hr>

        <h2 style="color: #003057;" id="entry"><i class="fas fa-brain"></i> Entry Details</h2 style="color: #003057;">
        <p>To be considered for the competition, please submit 1) subclass and optional cell type assignments for each query image series, 2) the tool (or a link to the tool) for computing and displaying these results, and 3) your name and contact information (see details below). This information must be directly included as part of <b><a href="#data_access">the data sheet below</a></b>.  Please download a copy of that file and submit at <strong>[LINK TO BE PROVIDED PRIOR TO DEADLINE]</strong> before the deadline, following the directions therein. All submissions must be received by <u><b>12:00pm (Pacific), on May 30th, 2025</b></u> to be considered for the competition.</p>
		
		<p>If you have any difficulties with submission or other clarification questions about the MapMySections data challenge, please <a href="mailto:rachel.hostetler@alleninstitute.org">contact Rachel Hostetler</a>.</p>
		
		<p>Participants will be scored based on two components, with a winner selected for each category independently:</p>
		<ol>
		<li><strong>Mapping accuracy</strong>: This winning algorithm for this category will be selected based on a quantitative assessment of accuracy of mapping to the whole mouse brain cell types from Yao et al 2023 included in the Allen Brain Cell Atlas at the "subclass" level of the hierarchy. Successful submissions will complete the mapping accuracy table that has one row for each test genetic tool and one column for each subclass found in VISp. Entries should represent the predicted fraction of cells mapping to a given subclass (e.g., rows sum to 1).</li>
		<li><strong>Utility of the tool</strong>: Creation of a user-friendly tool that allows users to input one or more query STPT image, apply the submitted mapping algorithm, and then receive the most likely cell type results as output. This tool will optionally connect output results back to the <a href="https://portal.brain-map.org/atlases-and-data/bkp/abc-atlas">Allen Brain Cell Atlas</a> or other Allen Brain Map knowledge of cell types in some way. The definition of tool is intentionally left open, but some examples could include a website, an interactive table, a downloadable software package, or an R Studio or Jupyter notebook. This winner will be selected by a panel of judges who will be grading on the following criteria: (1) usability (does it do something), (2) ease of use (can I figure out how to use it easily), (3) functionality (does the output make sense), and (4) sleekness (does it look cool and make you want to use it). The accuracy of mapping is not considered in this category.</li>
		</ol>
		
		<p><i><b>NOTE: </b></i>While not explicitly assessed for accuracy, <b><i>we encourage entrants to map to more refined levels of the cell type hierarchy (e.g., supertype, cluster) and/or to brain regions outside of VISp</b></i>. Broader mapping would improve the utility of the tool (second category) and would be considered in the unlikely event of a tie for mapping accuracy (first category).</p>  
		
		<p>Winners will have an opportunity to present their work at a to-be-scheduled webinar and will have their tool featured on Allen Brain Map <a href="https://portal.brain-map.org/">(brain-map.org)</a>. All entrants agree to have their submissions publicly linked at Allen Brain Map. Entrants may also be contacted for potential follow-up opportunities such as collaborative science or inclusion of tools in Allen Institute data work flows.</p>
		
		<br><hr>

        <h2 style="color: #003057;" id="data_access"><i class="fas fa-brain"></i> Primary image series data access</h2 style="color: #003057;">
        <p>Images are divided into a training data set with <b>396</b> genetic tools (where cell type assignments are provided) and a test data set with <b>100</b> genetic tools (where entrants are asked to predict cell type assignments). STPT image series and associated metadata are included for each genetic tool. This information is provided as a static spreadsheet (which doubles as the submission form) and as an interactive BioFileFinder (BFF) spreadsheet.</p>
		
		<h3><strong><a href="LINK TBD">Access the challenge submission sheet, including all challenge data.</a></strong></h3>
		
		<h3><strong><a href="LINK TBD">Access and interactive version of the data on BioFile Finder.</a></strong></h3>
		
		<p>In this spreadsheet, each row is a different genetic tool, and columns correspond to file locations, genetic tool metadata, and assessments of cell type classifications (details in the links above). Images are provided in two formats, which both may be useful for completing this challenge:</p>
		
		<ul>
		<li><strong>Raw STPT data</strong>: STPT data are collected as a series of images ("image series") that are stored in in OME-Zarr format. We provide both links to the underlying data and links for interactive visualizations of these image series using Neuroglancer in the above spreadsheets. See the <a href="#links">Useful links</a> section below for resources showing how to access and use these types of files.</li>
		<li><strong>CCF Registered Images</strong>: We have also registered most of the STPT image series to the Allen Common Coordinate Framework (CCF) with 25um voxel resolution, providing (1) a lower resolution summarization of fluorescence data, (2) a way of directly aligning images to cell type locations since MERFISH data is also aligned to the CCF, and (3) a way of determining which voxels in the image series correspond to primary visual cortex. See <b><a href="https://github.com/AllenInstitute/MapMySections/blob/main/ccf_and_registered_stpt_tutorial.ipynb">this Jupyter notebook</b></a> for details on how to access and use these data.</li>
		</ul>
		
		<p>The spreadsheet also includes multiple strategies for classifying cell types. Note that not all genetic tools were assessed using all of these strategies. </p>
		
		<ul>
		<li><strong>Target_Cell_Population</strong>: We include the cell populations targeted for all training experiments. <i><b>Note that this is not necessarily the cell populations that were actually identified.</b></i></li>
	    <li><strong>Qualitative Image Assessment</strong>: Most genetics tools include qualitative assessments for labeling strength, labeling density, and labeled cell populations made based on review of epifluorescence image data for the same genetic tool. In cases where more than one assessment was performed, all of the resulting assessments are included. While these calls do not constitute quantitative validation, they are likely to reflect actual cell type assignments in cases when SMART-Seq v4 data is missing.  Note that a value of "Neuron" indicates that the specific neuronal populations targeted could not be accurately assessed in that experiment.</li>
		<li><strong>Quantitative Cell Type Assignments</strong>: Cell types were also quantitatively assessed for many genetic tools using SMART-Seq v4. In these studies, up to 48 cells were collected from a given genetic tool and aligned to whole mouse brain cell types from the Allen Brain Atlas (links above) at the "subclass" level of the hierarchy. <b><i>These columns represent the ground truth data to be targeted by computational algorithms (more details in the spreadsheet).</b></i>. Numbers in these columns represent the count of cells from a given experiment that mapped to the corresponding subclass. While not provided, cells for some genetic tools were also aligned to supertype and cluster, and we encourage entrants to attempt such alignments as possible.</li>
		</ul>
		
		<br><hr>

        <h2 style="color: #003057;" id="spatial"><i class="fas fa-brain"></i> Spatial localization of mouse brain cell types</h2 style="color: #003057;">
        <p>The mouse whole brain transcriptomics cell type atlas contains 5,322 cell types (clusters) that are organized in a hierarchical manner with nested groupings of 34 classes, 338 subclasses, 1,201 supertypes and 5,322 clusters. The anatomical location of each cell type has been annotated using a comprehensive brain-wide MERFISH dataset with a total of ~4 million segmented and QC-passed cells, probed with a 500-gene panel and registered to the Allen Mouse Brain Common Coordinate Framework (CCF v3). The MERFISH data not only provide accurate spatial annotation of cell types at subclass, supertype and cluster levels, but also reveal fine-resolution spatial distinctions or gradients for cell types.</p>
		
		<p>In addition to gene tools image series, cell type information from this atlas <b><i>is required or recommended to succeed in this challenge</i></b>. The spatial location of every mouse cell type can be interactively explored using the Allen Brain Cell Atlas <a href="https://knowledge.brain-map.org/abcatlas#AQIBSzlKTjIzUDI0S1FDR0s5VTc1QQACSFNZWlBaVzE2NjlVODIxQldZUAADAAQBAAKDgDx7g46YHgOEuBCEhSrCAwQyTlFUSUU3VEFNUDhQUUFITzRQAAWBr6ZKgemsDoGggUeAktXoBgAHAAAFAAYBAQJGUzAwRFhWMFQ5UjFYOUZKNFFFAAN%2BAAAABAAACFZGT0ZZUEZRR1JLVURRVVozRkYACUxWREJKQVc4Qkk1WVNTMVFVQkcACgALAVRMT0tXQ0w5NVJVMDNEOVBFVEcAAjczR1ZURFhERUdFMjdNMlhKTVQAAwEEAQACIzAwMDAwMAADyAEABQEBAiMwMDAwMDAAA8gBAAAAAUFQOEpOTjVMWUFCR1ZNR0tZMUIAAlExTkNXV1BHNkZaMEROSVhKQlEAAwAEAQECgazlhIG8aWIDhMwEVIT%2BMccABQAGAQECRlMwMERYVjBUOVIxWDlGSjRRRQADfgAAAAQAAAhHNEk0R0ZKWEpCOUFUWjNQVFgxAAlMVkRCSkFXOEJJNVlTUzFRVUJHAAoACwFub25lAAJub25lAAMBBAEAAiMwMDAwMDAAA8gBAAUBAQIjMDAwMDAwAAPIAQAAAAIDAA%3D%3D">at this link</a>.</p>
		
		<p>Spatial data and associated metadata can be accessed for off-line use in the following locations:</p>
		
		<ul>
		<li><strong>MERFISH raw data images</strong>: Data from MERFISH experiments can be found in the <a href="https://doi.brainimagelibrary.org/doi/10.35077/g.610">Brain Image Library (BIL) here</a>. The relevant mouse has <a href="https://download.brainimagelibrary.org/aa/79/aa79b8ba5b3add56/638850/">ID=638850</a>.</li>
		<li><strong>CCF locations of mouse brain cells</strong>: Point-by-point mapping of the location of each cell in the original MERFISH coordinate space and the CCF <a href="https://alleninstitute.github.io/abc_atlas_access/descriptions/MERFISH-C57BL6J-638850-CCF.html"><strong>is available here</strong></a>. These data include cell type assignments for each cell, and this page also includes links to related data sets and to Jupyter notebooks for data access.</li>
		<li><strong>Code comparing cell types and anatomic structures</strong>: Specifically, <a href="https://alleninstitute.github.io/abc_atlas_access/notebooks/merfish_ccf_registration_tutorial.html">this Jupyter notebook</a> describes how to access and use the data above, relating CCF coordinates and mouse brain cell types.</li>
		<li><strong>Acronym list</strong>: <a href="https://brainmapportal-live-4cc80a57cd6e400d854-f7fdcae.divio-media.net/filer_public/c8/42/c8421c8e-72d8-4781-8505-8770e88671a7/whole_mouse_brain_acronym_june_2024.xlsx">This table</a> includes a list of acronyms and abbreviations for brain regions, genes, and cell types used in mouse whole brain cluster names, along with identifiers for relevant data sources (e.g., Mouse Brain Atlas, UBERON, CL).</li>
		</ul>
		
		<br><hr>
		
        <h2 style="color: #003057;" id="regions"><i class="fas fa-brain"></i> Restricting analysis to primary visual cortex (VISp)</h2 style="color: #003057;">
        <p><i><b>This challenge focuses only on cell types in primary visual cortex (VISp)</b></i>. A few resources are provided to help entrants constrain their data and associated analyses accordingly:</p>
		
		<ul>
		<li><strong>Image registration to Allen Common Coordinate Framework (CCF)</strong>: A large majority of STPT image series are registered to the CCF, meaning that every X-Y-Z location in STPT images can be assigned an X-Y-Z location (“voxel”) in a standard anatomical reference space along with an associated brain region name. Similarly, MERFISH image series on the <a href="https://portal.brain-map.org/atlases-and-data/bkp/abc-atlas">Allen Brain Cell Atlas</a> are also registered to the CCF, meaning each voxel also contains it’s cell type composition (from one mouse). More details on CCF registration and data access for STPT is listed above. CCF registration and data access for MERFISH data can be found in <a href="https://alleninstitute.github.io/abc_atlas_access/descriptions/MERFISH-C57BL6J-638850-CCF.html">Jupyter notebooks accompanying the Allen Brain Cell Atlas</a>.</li>
		<li><strong>Identifying subclasses in VISp</strong>: For the purposes of this challenge you can assume that all cells in VISp map to one of the subclasses listed in the "Quantitative Cell Type Assignments" columns described above. A list of subclass, brain region, and all other <b>abbreviations</b> can be found <a href="https://brainmapportal-live-4cc80a57cd6e400d854-f7fdcae.divio-media.net/filer_public/c8/42/c8421c8e-72d8-4781-8505-8770e88671a7/whole_mouse_brain_acronym_june_2024.xlsx">at this link</a>.
		<li><strong>Known spatial localization of cell types</strong>: The relationship between cell types and brain regions and cell types (including VISp) is described in detail in <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10719114/">Yao et al 2023</a>. These relationships can also be explored interactively using <a href="https://sea-ad.shinyapps.io/ACEapp/">Annotation Comparison Explorer</a> (“Mouse cell type classification” - “Spatial localization of brain cell types”).</li>
		</ul>
	
		
		<br><hr>
		
		<h2 style="color: #003057;" id="considerations"><i class="fas fa-brain"></i> Key considerations</h2 style="color: #003057;">
        <p>Some things to consider for this challenge: </p>
		<ol>
		<li>All images should have a single channel with signal. Signal in the channel with the least abundance can be ignored.</li>
		<li><b>Genetic tools can label multiple cell types!</b> In such genetic tools it is typically harder to identify labeled cell types than in ones when a single cell type is labeled.</li>
		<li>It is typically more challenging distinguishing between GABAergic interneurons subtypes than between glutamatergic and non-neuronal cell types.  For example, glutamatergic neurons tend to be highly localized by layer.</li>
		<li>Some genetics tools have weaker labeling than others.</li>
		<li>The Allen Brain Cell Atlas locations shown are cell centroids; however, genetic tools can label any cellular compartment.  The genetic tools in this challenge are cytoplasm reporters, which could potentially cause problems with dense labeling and cell overlap and make direct comparison a challenge.</li>
		</ol>
		
		<br><hr>

        <h2 style="color: #003057;" id="background"><i class="fas fa-brain"></i> Background</h2 style="color: #003057;">
        <p>The <a href="https://portal.brain-map.org/genetic-tools/genetic-tools-atlas">Genetics Tools Atlas</a> is a searchable web tool representing information and data on enhancer-adeno-associated viruses (enhancer AAVs) and mouse transgenes. The characterization data include Epifluorescence imaging (EPI), Serial Two-Photon Tomography (STPT), and Single Cell/Single Nucleus RNA-sequencing generated at the Allen Institute for Brain Science. These genetic tools have been published in several peer-reviewed manuscripts, and are largely available on <a href="https://www.addgene.org/collections/brain-armamentarium/">Addgene</a> (as part of the BRAIN Armamentarium Collection of plasmids and viral preps) or at <a href="https://www.jax.org/jax-mice-and-services/find-and-order-jax-mice/most-popular-jax-mice-strains/strain-collections">The Jackson Laboratory</a> (transgenic mice).</p>
		
		<p>The <a href="https://portal.brain-map.org/atlases-and-data/bkp/abc-atlas">Allen Brain Cell Atlas</a> provides a platform for visualizing multimodal single cell data across the mammalian brain and aims to empower researchers to explore and analyze multiple whole-brain datasets simultaneously. This open science resource, developed by the Allen Institute as part of the Brain Knowledge Platform, allows unprecedented insights into the enormous diversity of cell types in the brain and where they are. As the Allen Institute and its collaborators continue to add new modalities, species, and insights to the Allen Brain Cell Atlas, this groundbreaking platform will keep growing, opening up endless possibilities for discoveries and breakthroughs in neuroscience. The Allen Brain Cell Atlas enables the neuroscience community to identify more cell types in the brain, investigate the spatial location of cell types, investigate gene expression and co-expression patterns in cell types, and refine boundaries and knowledge of brain regions defined by gene expression.</p>
		
		<p>Allen Brain Map and BICCN have tools for allowing scientists to compare their gene expression data with cell types in motor cortex (<a href="https://azimuth.hubmapconsortium.org/">Azimuth</a>; accessible via the <a href="https://knowledge.brain-map.org/celltypes">Cell Type Knowledge Explorer</a>) and spanning the whole brain (<a href="https://portal.brain-map.org/atlases-and-data/bkp/mapmycells">MapMyCells</a>). These tools directly transfer (or “map”) cell type assignments to user-provided cells, also allowing a transfer of associated knowledge about that cell type. For example, cells in motor cortex matching to Pvalb interneurons based on gene expression are also likely to be fast spiking and to have either basket or chandelier morphologies. Similarly, putative spatial locations can be assigned to a cell collected from a dissociation protocol by mapping it to the whole mouse brain and then seeing where that cell is located using the <a href="https://portal.brain-map.org/atlases-and-data/bkp/abc-atlas">Allen Brain Cell Atlas</a>. Finally, by mapping cell type assignments to cells collected from individuals with Alzheimer’s disease (e.g., in <a href="https://portal.brain-map.org/explore/seattle-alzheimers-disease">SEA-AD</a>) and other disorders, scientists can understand which cell types change and how they change in different states. While other algorithms exist for mapping transcriptomics (and other ‘-omics’) data between reference cell types and query data, Azimuth and MapMyCells are web-based, making them accessible to researchers and students with a wide range of computational and biological competencies. Last year’s data challenge, <a href="https://alleninstitute.org/events/mapmyspikes/">MapMySpikes</a>, asked participants to accurately make the connection between firing properties of a cell and the cell’s molecular type to relate it to existing knowledge of cell types on Allen Brain Map; otherwise, nearly all web-based mapping algorithms focus on gene expression. MapMySections, like MapMySpikes, aims to extend the types of experimental data that can be aligned to transcriptomically-defined cell types.</p>
		
		<br><hr>

        <h2 style="color: #003057;" id="links"><i class="fas fa-brain"></i> Useful links</h2 style="color: #003057;">
		
		<p><strong>Working with large image files:</strong></p>
		<ul>
		<li><a href="https://cloudpathlib.drivendata.org/stable/">This python library</a> provides a convenient way to access data via an s3 bucket.</li>
		<li><a href="https://community.brain-map.org/t/tutorial-finding-aavs-for-targeting-cholinergic-cell-populations-in-the-striatum/3673">This Allen Brain Map Community Forum post</a> describes some basic Neuroglancer functionality.</li>
		<li><a href="https://alleninstitute.github.io/microns_tutorial/em_04_neuroglancer.html">This MICrONs tutorial</a> goes into a bit more depth about how to use Neuroglancer.</li>
		<li><a href="https://link.springer.com/article/10.1007/s00418-023-02209-1">This open-access manuscript</a> describes the OME-Zarr format in detail. </li>
		</ul>

		<p><strong>Strategies for aligning tissues sections to the Allen CCF:</strong></p>
		<ul>
		<li><a href="https://community.brain-map.org/t/mouse-brain-reference-atlases-mapping-your-own-data/243">This Allen Brain Map <strong>Community Forum post</strong></a> lists several community tools (as of 2019) for mapping images to our reference atlas.</li>
		<li>A subset of this more recent (2023) <a href="https://portal.brain-map.org/explore/toolkit/community-tools/computational-tools/data-registration-tools">list of <strong>Community Tools</strong></a> related to computational image registration to CCF.</li>
		</ul>
		
		<p><strong>Atlases related to this challenge:</strong></p>
		<ul>
		<li><strong>Genetics Tools Atlas</strong>: <a href="https://portal.brain-map.org/genetic-tools/genetic-tools-atlas">https://portal.brain-map.org/genetic-tools/genetic-tools-atlas</a></li>
		<li><strong>Allen Brain Cell Atlas</strong>: <a href="https://portal.brain-map.org/atlases-and-data/bkp/abc-atlas">https://portal.brain-map.org/atlases-and-data/bkp/abc-atlas</a></li>
		<li><strong>MapMyCells</strong>: <a href="https://portal.brain-map.org/atlases-and-data/bkp/mapmycells">https://portal.brain-map.org/atlases-and-data/bkp/mapmycells</a></li>
		</ul>
		
		<p><strong>Other relevant links:</strong></p>
		<ul>
		<li><a href="https://portal.brain-map.org/help-and-community/guide-cell-types"><strong>Cell Type Support</strong></a>: An exhaustive list of Allen Institute's resources to help better understand cell types.
		<li><a href="https://alleninstitute.org/events/mapmyspikes/"><strong>MapMySpikes</strong></a>: This was last year's data challenge focused on using electrophysiology data to define transcriptomic cell types.</li>
		<li><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11195086/"><strong>Scientific Preprint</strong></a> highlighting the genetic tools included in this data challenge.</li>
		</ul>

    </div>

</body>
</html>
